tesseract-server/src/app.rs

```diff
@@ -9,21 +9,52 @@ use crate::db_config::Database;
 use crate::handlers::{
     aggregate_handler,
     aggregate_default_handler,
+    flush_handler,
     index_handler,
     metadata_handler,
     metadata_all_handler,
 };
 
+use std::sync::{Arc, RwLock};
+
+/// Represents the location of a local schema file.
+pub struct LocalSchema {
+    filepath: String,
+}
+
+/// Represents the location of a remote schema file.
+pub struct RemoteSchema {
+    endpoint: String,
+}
+
+/// Holds data about the source of a schema file.
+#[derive(Debug, Clone)]
+pub enum SchemaSource {
```

Comment:
the records `LocalSchema` and `RemoteSchema` only need to be declared inside the enum, the struct declarations are not needed in this case.